### ISSUE TITLE
2.7.1 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2611]
## Merged PRs
- #2575 
- #2586 
- #2605
- #2593 
- #2555

## Pick list
```
git cherry-pick eabaac5f1ab02239196fa324a64bab7b3986c791
git cherry-pick aa4c242011326891223fb3aaade21b5b998efa42
git cherry-pick 56d68a2cde06622e7748291485a6c4a81e731153
git cherry-pick a1d40a5f65225ea287d9ea75686302056cf9d3f3
git cherry-pick 15372b88dec5a0b2ea2bba445aff2baecac75539
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.7.0..HEAD --pretty='format:- %s' `
```
- Fixes #2524 Author Entity Autolabel Length (#2555)
- Update drupal/smart_date requirement from 4.0.2 to 4.0.3 (#2593)
- Closes #2604 Change port number in az_mail module from 587 to 2587 (#2605)
- Update drupal/linkit requirement from 6.0.0-rc1 to 6.0.0 (#2586)
- Update drupal/core-recommended requirement from 10.0.9 to 10.0.10 (#2575)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.1
